packages/components/src/item-group/stories/index.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { ItemGroup } from '../item-group/component';
+import { Item } from '../item/component';
+import type { ItemProps } from '../types';
+
+const meta: ComponentMeta< typeof ItemGroup > = {
+	component: ItemGroup,
+	title: 'Components (Experimental)/ItemGroup',
+	subcomponents: { Item },
+	argTypes: {
+		as: { control: { type: null } },
+		children: { control: { type: null } },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const mapPropsToItem = ( props: ItemProps, index: number ) => (
+	<Item { ...props } key={ index } />
+);
+
+const Template: ComponentStory< typeof ItemGroup > = ( props ) => (
+	<ItemGroup { ...props } />
+);
+
+export const Default: ComponentStory< typeof ItemGroup > = Template.bind( {} );
```

Comment:
Previously, `ItemGroup`'s stories were much more complex, and involved many other components in an effort to showcase how the component could be used:
- in combination with `Dropdown`
- for more "complex" layouts (i.e. the ones used in Global Styles sidebar)

For now, I've decided not to convert those examples to TypeScript/controls because I feel like we've been recently trying to keep Storybook examples closer to the component they represent.

What do folks think?

Reply:
Yes, I agree the previous stories were a bit much.

Though, I wasn't familiar with this component and my first impression when seeing the new default story was that I couldn't understand what the component was supposed to be used for. Would it be more straightforward if we just had a group of three clickable `Item`s? I'm assuming that is the main use case because I had a hard time finding an example in the Gutenberg app where there were unclickable items.

And if we want to highlight other usages like unclickable items, different padding sizes, borders, etc. we could do that in separate stories (doesn't have to be in this PR).

Reply:
Pushed some changes in https://github.com/WordPress/gutenberg/pull/46945/commits/6623d9b114a251d9d32b76dd8be5e90081dd60b7 — let me know if they look good to you!